modules/deploy/partials/high-availability.adoc

```diff
@@ -391,7 +393,9 @@ Redpanda requires a minimum of three availability zones when using a multi-AZ ap
 
 === Multi-AZ optimization
 
-You can configure follower fetching to help ease the latency problems associated with a multi-AZ configuration. This is achieved by configuring consumers to advertise their preferred rack using the `client.rack` option within their consumer configuration. This allows consumers to read data from their closest replica rather than always reading from a (potentially non-local) partition leader. Using this capability has the additional benefit of reducing non-local network traffic and thereby minimizing any cross-AZ network charges.
+You can configure follower fetching to help ease the cross-AZ cost problems associated with a multi-AZ configuration. This is achieved by configuring consumers to advertise their preferred rack using the `client.rack` option within their consumer configuration. This allows consumers to read data from their closest replica rather than always reading from a (potentially non-local) partition leader.
+
+Enabling follower fetching has the additional potential benefit of reducing broker latency. If the nearest replica is lagging, however, you may instead encounter increased data latency. Make sure to monitor your system to determine if the cost savings are worth this data latency risk.
```

Comment:
I'd drop this second paragraph "enabling follower fetching...", since the benefit of lower broker latency is pretty small when compared with the disbenefit of potentially reading from a straggling broker. I'd leave it at a cost optimisation I think?

Reply:
@pmw-rp Do you think it's worth mentioning the risk of higher latency? That was the main purpose of the second paragraph, to explain that there's a potential downside to the cost savings.